src/pages/service.tsx

```diff
@@ -1,24 +1,29 @@
 import React, { ReactElement } from 'react';
 import { NextPageContext } from 'next';
 import { parseCookies } from 'nookies';
+import { ErrorInfo } from '../types';
+import FormElementWrapper from '../components/FormElementWrapper';
 import Layout from '../layout/Layout';
 import { OPERATOR_COOKIE, SERVICE_COOKIE, PASSENGER_TYPE_COOKIE } from '../constants';
-import { deleteCookieOnServerSide } from '../utils';
 import { getServicesByNocCode, ServiceType } from '../data/auroradb';
+import ErrorSummary from '../components/ErrorSummary';
 
 const title = 'Service - Fares Data Build Tool';
 const description = 'Service selection page of the Fares Data Build Tool';
+const errorId = 'service-error';
 
 type ServiceProps = {
     operator: string;
     passengerType: string;
     services: ServiceType[];
+    error: ErrorInfo[];
 };
 
-const Service = ({ operator, passengerType, services }: ServiceProps): ReactElement => (
+const Service = ({ operator, passengerType, services, error }: ServiceProps): ReactElement => (
     <Layout title={title} description={description}>
         <main className="govuk-main-wrapper app-main-class" id="main-content" role="main">
             <form action="/api/service" method="post">
+                <ErrorSummary errors={error} />
                 <div className="govuk-form-group">
```

Comment:
I think this line would need to be <div className={`govuk-form-group ${error.length > 0 ? 'govuk-form-group--error' : ''}`}> to get the red line alongside it

Reply:
Added, thanks!